src/main/java/bio/terra/workspace/service/resource/controlled/flight/create/CreateControlledResourceFlight.java

```diff
@@ -82,16 +85,22 @@ public CreateControlledResourceFlight(FlightMap inputParameters, Object beanBag)
                 flightBeanBag.getCrlService(),
                 flightBeanBag.getWorkspaceService(),
                 resource.castToAiNotebookInstanceResource()),
-            notebookGcpRetryRule);
+            gcpRetryRule);
         addStep(
             new CreateAiNotebookInstanceStep(
                 flightBeanBag.getCrlService(),
                 resource.castToAiNotebookInstanceResource(),
                 flightBeanBag.getWorkspaceService()),
-            notebookGcpRetryRule);
+            gcpRetryRule);
+
         // TODO(PF-469): Set permissions on service account and notebook instances.
         break;
       case BIG_QUERY_DATASET:
+        addStep(
```

Comment:
Maybe a comment here that the permissions are set in the create step, since it will look irregular.